### ISSUE TITLE
Dimension out of range with Mask

### ISSUE THREAD

**Issue by AdrienTasicSC** [flowtyone/ComfyUI-Flowty-TripoSR#38]
I'm trying to plug a contrasted depth-map - resized to 512*512 - on the _reference_mask_ input but got this error:

`Dimension out of range (expected to be in range of [-2, 1], but got 2)`

**Comment by jokroese** [comment]
I'm getting a similar error when I try to use mask.

Here's the workflow I'm using: [2d-to-3d-triposr.json](https://github.com/flowtyone/ComfyUI-Flowty-TripoSR/files/14776326/2d-to-3d-triposr.json) which looks like this:

<img width="863" alt="Screenshot 2024-03-27 at 17 27 13" src="https://github.com/flowtyone/ComfyUI-Flowty-TripoSR/assets/7649772/1de1b124-d88b-4cf9-8abf-dfcf24e07c21">

and the full error I get is:

```
Error occurred when executing TripoSRSampler:

Sizes of tensors must match except in dimension 2. Expected size 768 but got size 64 for tensor number 1 in the list.

  File "/Users/jo/Documents/dev/ComfyUI/execution.py", line 151, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jo/Documents/dev/ComfyUI/execution.py", line 81, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jo/Documents/dev/ComfyUI/execution.py", line 74, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jo/Documents/dev/ComfyUI/custom_nodes/ComfyUI-Flowty-TripoSR/__init__.py", line 88, in sample
    image = torch.cat((image, mask), dim=2).detach().cpu().numpy()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
